neptune_tf_plugin/__init__.py

```diff
@@ -13,3 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+import click
+
+
+@click.command('tf')
+@click.option('--api-token', '-a', help='Neptune Authorization Token')
+@click.option('--project', '-p', help='Project name')
+@click.argument('path', required=True)
+def sync(api_token, project, path):
```

Comment:
I dont like shadowing sync name. This can lead to issues. import sync as anoter name?